web/modules/custom/hoeringsportal_citizen_proposal/src/Helper/Helper.php

```diff
@@ -248,6 +321,20 @@ private function getProposalStorage(): PrivateTempStore {
     return $this->tempStoreFactory->get('hoeringsportal_citizen_proposal');
   }
 
+  /**
+   * Determine if a proposal has enough votes.
+   *
+   * @param int $supportCount
+   *   The support a proposal has received.
+   *
+   * @return bool
+   *   Whether the proposal has enough votes.
+   */
+  private function hasEnoughVotes(int $supportCount): bool {
+    // @todo Use constant to compare against when it is merged.
+    return !($supportCount < 10);
```

Comment:
This has been merged now – and given rise to a merge conflict …

Reply:
Resolved merge conflict